x-pack/filebeat/module/cisco/_meta/docs.asciidoc

```diff
@@ -16,17 +16,6 @@ Cisco ASA devices also support exporting flow records using NetFlow, which is
 supported by the {filebeat-ref}/filebeat-module-netflow.html[netflow module] in
 {beatname_uc}.
 
-[WARNING]
```

Comment:
Did you run `make update` after removing this block? The doc build is failing because `cisco.asciidoc` still has a reference to the `dynamic-script-compilations` anchor.

Reply:
Ahh that's where it was.  Removed.